apps/api/tsconfig.app.json

```diff
@@ -4,7 +4,9 @@
     "outDir": "../../dist/out-tsc",
     "types": ["node"],
     "emitDecoratorMetadata": true,
-    "target": "es2021"
+    "target": "es2021",
+    "module": "commonjs",
+    "moduleResolution": "node"
```

Comment:
This is part of [tsconfig.base.json](https://github.com/ghostfolio/ghostfolio/blob/main/tsconfig.base.json#L7), why do you set it again?

Reply:
Yep, you're right. That's not necessary. I will push a commit and remove it again.